### PR TITLE
Deprecate `existingDataSeedInstance` option to CREATE DATABASE

### PR DESCRIPTION
**Pull request by mnd999**
Replace with new option `existingDataSeedServer`

Documenting https://github.com/neo-technology/neo4j/pull/27123

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1047.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1047.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by HannesSandberg — APPROVED** [review]

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -16,6 +16,44 @@ New features are added to the language continuously, and occasionally, some feat
...
+
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE db OPTIONS { existingDataSeedInstance: ... }
+----
+| The `CREATE DATABASE` option `existingDataSeedInstance` has been deprecated and replaced with the option `existingDataSeedServer`. The functionality is unchanged.
+|===
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
=== Deprecated features

[cols="2", options="header"]
|===
| Feature
| Details

a|
label:functionality[]
label:deprecated[]
[source, cypher, role="noheader"]
----
CREATE DATABASE db OPTIONS { existingDataSeedInstance: ... }
----
| The `CREATE DATABASE` option `existingDataSeedInstance` has been deprecated and replaced with the option link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/create-database/#manage-databases-create-database-options[`existingDataSeedServer`]. The functionality is unchanged.
|===

=== New features

[cols="2", options="header"]
|===
| Feature
| Details

a|
label:functionality[]
label:new[]
[source, cypher, role="noheader"]
----
CREATE DATABASE db OPTIONS { existingDataSeedServer: ... }
----
| The option link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/standard-databases/create-database/#manage-databases-create-database-options[`existingDataSeedServer`] has been added to `CREATE DATABASE`. The functionality is the same as the deprecated option `existingDataSeedServer`, which this replaces.
|===

```

**Review by JPryce-Aklundh — APPROVED** [review]
Looks good - I just reversed order of New and Deprecated and added links.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1047.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1047.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**